packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md

```diff
@@ -0,0 +1,146 @@
+
+Client-based rate limiting can be helpful in the following scenarios:
+
+* You have several OAuth applications that are split across multiple teams. You need to ensure that one OAuth application doesn't consume all of the assigned per minute request limits.
+* You want the ability to isolate any rogue OAuth applications from other valid OAuth applications.
+* You want to ensure that every application team adheres to best-coding practices such as proper error handling without resulting in redirect loops.
+* You want protection against random synthetic tests or batch jobs that make numerous requests every minute in production.
+
+### How it works
+
+The best way to describe how client-based rate limiting works is to provide some example use cases.
+
+#### Client-based isolation for a unique IP address
+
+Bob and Alice are working from home and have distinct IP addresses. Both Bob and Alice try to sign in to their company's portal application (clientID: portal123) at the same time. When they make the authorize request to `https://company.okta.com/oauth2/v1/default/authorize?clientId=portal123`, the client-based rate-limiting framework creates a unique per minute request quota for both Bob and Alice using their IP address and the OAuth client ID that they are trying to access.
+
+* Bob: (IP1 + portal123) Gets a quota of 60 total requests per minute and a maximum of two concurrent requests
+* Alice: (IP2 + portal123) Gets a quota of 60 total requests per minute and a maximum of two concurrent requests
+
+Let's assume the org-wide quota for the `/authorize` endpoint is a total of 2,000 requests per minute on Custom Authorization Servers. For some reason, Bob decides to run a batch job that triggers about 2,000 `/authorize` requests per minute. Without the client-based rate-limiting framework, Bob would consume the entirety of the total allowed requests per minute (2,000 requests per minute). This would result in HTTP 429 errors for both Bob and Alice, which makes the application inaccessible for everyone.
+
+With the client-based rate-limiting framework enabled, after Bob exceeds his individual limit of 60 requests per minute, HTTP 429 errors are sent. Requests that originate from Bob's IP address, and the specific device from which the requests are made, start receiving the errors. Alice could continue to access the application without any issues.
```

Comment:
With the client-based rate-limiting framework enabled, after Bob exceeds his individual limit of 60 requests per minute, any requests that originate from his IP address and the specific device from which the requests are made generate HTTP 429 errors.